test/cqlpy/test_describe.py

```diff
@@ -476,31 +476,47 @@ def test_desc_index(cql, test_keyspace):
         create_idx_c = f"CREATE INDEX named_index ON {tbl}(c)"
         # Only Scylla supports local indexes
         has_local = is_scylla(cql)
+
+        # Cassandra inserts a space between the table name and parentheses,
+        # Scylla doesn't. This difference doesn't matter because both are
+        # valid CQL commands
+        # Scylla doesn't support sai custom class.
+        if is_scylla(cql):
+            maybe_space = ''
+            custom_class = 'dummy-vector-backend'
+        else:
+            maybe_space = ' '
+            custom_class =  'sai'
+
+
         if has_local:
             create_idx_ab = f"CREATE INDEX ON {tbl}((a), b)"
+        create_idx_d = f"CREATE INDEX custom ON {tbl}(c) USING '{custom_class}'"
+        create_idx_e = f"CREATE CUSTOM INDEX custom1 ON {tbl}(b) USING '{custom_class}'"
 
         cql.execute(create_idx_b)
         cql.execute(create_idx_c)
+        cql.execute(create_idx_d)
+        cql.execute(create_idx_e)
         if has_local:
             cql.execute(create_idx_ab)
 
         b_desc = cql.execute(f"DESC INDEX {test_keyspace}.{tbl_name}_b_idx").one().create_statement
         if has_local:
             ab_desc = cql.execute(f"DESC INDEX {test_keyspace}.{tbl_name}_b_idx_1").one().create_statement
         c_desc = cql.execute(f"DESC INDEX {test_keyspace}.named_index").one().create_statement
+        d_desc = cql.execute(f"DESC INDEX {test_keyspace}.custom").one().create_statement
+        e_desc = cql.execute(f"DESC INDEX {test_keyspace}.custom1").one().create_statement
 
-        # Cassandra inserts a space between the table name and parentheses,
-        # Scylla doesn't. This difference doesn't matter because both are
-        # valid CQL commands
-        if is_scylla(cql):
-            maybe_space = ''
-        else:
-            maybe_space = ' '
+
+        
         assert f"CREATE INDEX {tbl_name}_b_idx ON {tbl}{maybe_space}(b)" in b_desc
         if has_local:
             assert f"CREATE INDEX {tbl_name}_b_idx_1 ON {tbl}((a), b)" in ab_desc
         
         assert f"CREATE INDEX named_index ON {tbl}{maybe_space}(c)" in c_desc
+        assert f"CREATE CUSTOM INDEX custom ON {tbl}{maybe_space}(c) USING '{custom_class}'" in d_desc
+        assert f"CREATE CUSTOM INDEX custom1 ON {tbl}{maybe_space}(b) USING '{custom_class}'" in e_desc
```

Comment:
I'd like to see some more tests for the validation logic of CREATE (CUSTOM) INDEX. For example:
- USING custom_class but with CREATE INDEX and not CREATE CUSTOM INDEX
- Passing custom class that is not supported by Scylla
- CUSTOM index without any custom class being provided